lib/stdout-manipulator.js

```diff
@@ -64,10 +64,17 @@ function manipulate(buffer) {
 
 function detectState(lines) {
   const clearLines = removeColors(lines);
-  const compilationError = clearLines.some(line => compilationCompleteWithErrorRegex.test(line));
+  const newCompilation = clearLines.some(line => newCompilationRegex.test(line));
+  const compilationError = clearLines.some(
```

Comment:
In the original code the test was done on the `compilationCompleteWithErrorRegex` and it was enough to detect error, why did you add the `typescriptErrorRegex` and `typescriptPrettyErrorRegex` to the test?

Reply:
Because in older versions of TypeScript `compilationCompleteWithErrorRegex ` does not exist. Se my comment below